lib/upgrade/etcd/conversionv1v3/rules_def.go

```diff
@@ -33,9 +33,9 @@ var numProtocol1 = numorstring.ProtocolFromInt(240)
 var icmpType1 = 100
 var icmpCode1 = 200
 
-var cidr1StrictMaskStr = "10.0.0.0/24"
+var cidr1StrictMaskStr = "192.168.0.128/25"
```

Comment:
what was the purpose of this change?

Reply:
`192.168.0.128/25` is strictly masked (normalized). We use this in Profile and Policy conversion tests where `192.168.0.129/25` is passed as APIv1 and we expect this strictly masked out from the v3 output